### DISCUSSION TITLE
Download hangs and never completes.

### DISCUSSION THREAD

**Discussion by SileE-NC** [aclap-dev/video-downloadhelper#1052] [Help and Q&A]
### Your question

I've been trying to download videos from 2 different sites but the download won't complete. It stops somewhere between 10% and 40% done and then... nothing. I've downloaded from both sites before without an issue.
Thanks for your help.

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"75ef0249a5d54689"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:128.0) Gecko/20100101 Firefox/128.0
{
    "valid_license_message_has_been_displayed": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 10
}

**Comment by GrampaWildWilly** [comment]
But the mystery is still a mystery . . . until you give us URLs of web pages on which these uncooperative videos reside.

**Comment by SileE-NC** [comment]
Sorry for the delay in answering. I did some more problem solving myself
and have figured some things out. I downloaded the app again on my laptop
and, rather than right-clicking and accessing downloadhelper that way, I
now use the icon in the bar across the top of the screen (can't remember
what that's called.) It doesn't work at home, which is a pain, but it does
work where there's a better Internet service than ours (we're in the blue
ridge mountains and our only choice is crummy satellite service) but right
down the road the local library has better service.

Thanks for following up with my original request for help. I appreciate
downloadhelper so much!

Síle Ellison-Níchionna
